dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/program/notification/ProgramNotificationListener.java

```diff
@@ -44,30 +40,8 @@
 public class ProgramNotificationListener {
   private final ProgramNotificationService programNotificationService;
 
-  private final TrackerNotificationWebHookService trackerNotificationWebHookService;
-
   @TransactionalEventListener(fallbackExecution = true)
   public void onEnrollment(ProgramEnrollmentNotificationEvent event) {
```

Comment:
@teleivo This is still used somewhere down the line in the SMS part.
It should be remove when we fix it there